StatisticsTests/Distributions/ParametricBootstrap.cs

```diff
@@ -21,7 +21,8 @@ public void Bootstrap_Works_with_Reasonable_RandomRange(double min, double max,
                 randyPacket[i] = ((double)i +0.5) / (double)dist.SampleSize;
             }
             Statistics.IDistribution bootstrap = dist.Sample(randyPacket);
-            Assert.Equal(.5, bootstrap.Mean);
+            Statistics.Distributions.Uniform bu = (Statistics.Distributions.Uniform)bootstrap;
```

Comment:
may we please use bootstrappedUniform instead of bu?

Reply:
yes feel free to change.